client/src/components/IframePreview.jsx

```diff
@@ -30,18 +30,20 @@ const IframePreview = forwardRef(({ slug, isLoading, device = 'iPhone X' }, ref)
     <iframe
       ref={iframeRef}
       src={`${import.meta.env.VITE_BACKEND_URL}/site/${slug}/`}
-      className="w-full h-full border-0"
+      className={`w-full h-full border-0  ${device === "iPhone X" && "pt-8 bg-black"}`}
     ></iframe>
   ) : (
     <div className="w-full h-full flex items-center justify-center text-gray-400">
       No preview available
     </div>
   );
 
+
+
   return (
     <div className="w-full h-full flex items-center justify-center">
       <div className="scale-[0.7]">
-        <DeviceFrameset device={device}>
+        <DeviceFrameset device={device}  >
```

Comment:
extra space due to running formatter or added by mistake?

Reply:
this was by mistake